### PR TITLE
API-5159: Appointment documentation

### PR DESCRIPTION
**Pull request by jjabando1**

### CODE REVIEW

File: data-query/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <properties>
     <datamart-starter.version>2.0.1</datamart-starter.version>
     <fall-risk.version>1.0.29</fall-risk.version>
-    <fhir-resources.version>5.0.20</fhir-resources.version>
+    <fhir-resources.version>6.0.2</fhir-resources.version>
```
Comment (ilaflamme-va) [inline data-query/pom.xml:16]: What changed in this major version upgrade? Nothing affecting us?

Comment (jjabando1) [reply data-query/pom.xml:16]: R4 AppointmentApi

Comment (ilaflamme-va) [reply data-query/pom.xml:16]: the _major_ version `5.x.x` -> `6.x.x` that usually indicates something major.

Comment (jjabando1) [reply data-query/pom.xml:16]: So it looks like its R4 simpleresource -> resource: https://github.com/department-of-veterans-affairs/health-apis-fhir-resources/commit/7ba44b6eff68269a799dedb682d00a40203428d4

File: data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java [hunk 2]
```diff
@@ -34,6 +35,7 @@
             name = "OauthFlow",
             scopes = {
               "patient/AllergyIntolerance.read",
+              "patient/Appointment.read",
```
Comment (ilaflamme-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java:38]: "system/Appointment.read`

File: data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java [hunk 3]
```diff
@@ -86,6 +88,9 @@
                       @OAuthScope(
                           name = "patient/AllergyIntolerance.read",
                           description = "read allergy intolerances"),
+                      @OAuthScope(
+                          name = "patient/Appointment.read",
```
Comment (ilaflamme-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java:92]: `system/Appointment.read`

**Review by ilaflamme-va — APPROVED** [review]

**Pull request merged**